### ISSUE TITLE
Should tissue be child of 'object aggregate'?

### ISSUE THREAD

**Issue by linikujp** [OmniSearch/ncro#23]
I think tissue maybe just child of 'anatomical structure'.

**Comment by alanruttenberg** [comment]
It is already so, in the branch.

**Comment by linikujp** [comment]
Sorry, the title was misleading.
Let me rephrase it.
Currently, tissue is both child of 'object aggregate' and 'anatomical structure'.
Should tissue have only one parent term? And I think it should be 'anatomical structure'.

**Comment by alanruttenberg** [comment]
I agree. In any case I don't use the object, object aggregate, and fiat object part terms in ontologies I edit as they are not univocal.
